nixos/modules/services/misc/nix-daemon.nix

```
@@ -560,15 +560,17 @@ in
     environment.etc."nix/machines" =
       { enable = cfg.buildMachines != [];
         text =
-          concatMapStrings (machine:
-            "${if machine.sshUser != null then "${machine.sshUser}@" else ""}${machine.hostName} "
-            + (if machine.system != null then machine.system else concatStringsSep "," machine.systems)
-            + " ${if machine.sshKey != null then machine.sshKey else "-"} ${toString machine.maxJobs} "
+          concatMapStrings (machine: let
+            concatComma = lst: concatStringsSep "," lst;
```

```suggestion
            concatComma = concatStringsSep ",";
```

Because what's an L.S.T.?